modules/apps/frontend-theme/frontend-theme-classic-style-guide-sample-web/src/main/resources/META-INF/resources/css/edu.scss

```diff
@@ -0,0 +1,3 @@
+.sheet {
+    background-color: red!important;
+}
```

Comment:
We need to create a new `file.scss` in a module (**not** `_file.scss` since we need to "export" it)